core/src/main/java/org/bitcoin/crawler/PeerSeedProtos.java

```diff
@@ -3,6 +3,7 @@
 
 package org.bitcoin.crawler;
 
+@SuppressWarnings("ALL")
```

Comment:
You mean "no solution"? My explanation is in the commit's title: these classes are generated, it does not make any sense to deal with warnings generated for these.